charts/stfc-cloud-longhorn/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: longhorn
+version: 1.0.0
+dependencies:
+# https://github.com/longhorn/charts/releases
+- name: longhorn
+  version: 1.7.1
```

Comment:
1.8.0 is now the latest chart

Reply:
I'm aware - but this is a copy-paste of the current chart we're using - I don't want to start adding changes here to something we know works